parkhangs-webapp/src/components/index.css

```diff
@@ -3,3 +3,31 @@ body {
   padding: 0;
   font-family: sans-serif;
 }
+
```

Comment:
why is all this stuff in the index.css file when it doesn't have anything to do with the app level css

Reply:
Moved to new css file